javav2/example_code/cloudfront/Readme.md

```diff
@@ -4,67 +4,173 @@ This README discusses how to run and test the Java code examples for Amazon Clou
+|-----------------------------|------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CreatePublicKey             | `publicKeyFileName`                | Put a copy of your public key file in `src/main/resources` directory. <br/>The public key is the counterpart to the private key you use to sign.<br/>See the [CloudFront Developer Guide](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-trusted-signers.html#private-content-creating-cloudfront-key-pairs). |
+| CreateKeyGroup              |                                    |                                                                                                                                                                                                                                                                                                                                                   |
+| CreateOriginAccessControl   |                                    |                                                                                                                                                                                                                                                                                                                                                   |
+| CreateDistribution          |                                    |                                                                                                                                                                                                                                                                                                                                                   |
+| CreateBucketPolicy          |                                    | Creates and uploads the bucket policy.                                                                                                                                                                                                                                                                                                            |
+| CreateCannedPolicyRequest   | `privateKeyFullPath`               | `privateKeyFullPath` is the full path to private key whose public key is used by the CreatePublicKey file.                                                                                                                                                                                                                                        |
```

Comment:
`privateKeyFullPath` is the full path to the private key. This goes with the public key that's used by the CreatePublicKey file.